docs/docs/install/requirements.md

```diff
@@ -8,18 +8,20 @@ Hardware and software requirements for Immich:
 
 ## Software
 
-- [Docker](https://docs.docker.com/get-docker/)
-- [Docker Compose](https://docs.docker.com/compose/install/)
+Immich requires [**Docker**](https://docs.docker.com/get-started/get-docker/) with the **Docker Compose plugin**:
+
+- **Docker Engine**: This CLI variant is suitable for Linux servers or Windows via WSL2.
+- **Docker Desktop**: This GUI variant is suitable for Linux (desktop), Windows or macOS.
+
+The Compose plugin will be installed by both Docker Engine and Desktop by following the linked installation guides; it can also be [separately installed](https://docs.docker.com/compose/install/) (pick the plugin, not the standalone).
 
 :::note
-Immich requires the command `docker compose` - the similarly named `docker-compose` is [deprecated](https://docs.docker.com/compose/migrate/) and is no longer compatible with Immich.
+Immich requires the command `docker compose` (the plugin); the similarly named `docker-compose` (the standalone) is [deprecated](https://docs.docker.com/compose/migrate/) and is no longer compatible with Immich.
```

Comment:
I think this plugin vs standalone talk is more confusing, not less

Reply:
For immich’s purposes strictly, it is; however, the [Docker Compose installation](https://docs.docker.com/compose/install/) docs that this page links specifically give the choice between installing Docker Desktop, the Compose plugin, or the Compose standalone.

Reply:
It gives the choice but also pretty clearly notes that the standalone is deprecated, so I don't think we need to repeat that here.

Reply:
That’s fair yeah; I just figured people might want to join the dots on which installation option corresponds to which command, as an explanation for why the old command doesn’t work. I did delete the reference to “v2” over “v1” since I could hardly find it on Docker’s website. But yeah, omitting those brackets is fine.

Reply:
> That’s fair yeah; I just figured people might want to join the dots on which installation option corresponds to which command, as an explanation for why the old command doesn’t work. I did delete the reference to “v2” over “v1” since I could hardly find it on Docker’s website. But yeah, omitting those brackets is fine.

done

> I'd suggest replacing `compatible` with `supported`. New Synology nas systems use docker-compose and it seems to be running fine for me.

and done